examples/finnegans_wake_demo/finnegans-wake-demo.py

```diff
@@ -102,7 +102,7 @@
                      expiration=policy_end_datetime)
 
 assert policy.public_key == policy_pubkey
-policy.publishing_mutex.block_until_complete()
+policy.treasure_map_publisher.block_until_complete()
```

Comment:
This name change from "publishing_mutex" to "treasure_map_publisher" is such a huge improvement by itself.  I already love this PR.
💯